services/beaconchain/src/main/java/tech/pegasys/artemis/services/beaconchain/BeaconChainService.java

```diff
@@ -105,6 +105,16 @@ public void init(ServiceConfig config) {
               config.getConfig().getIdentity(),
               config.getConfig().getBootnodes(),
               config.getConfig().isBootnode());
+
+      // TODO - issue #827:
+      //      Once i have a reliable way to be notified when
+      //      libp2p peers are found then this can be removed
```

Comment:
I don't totally understand what the effect of sleeping here will be, but we shall see